src/core/vectorlayercounterpart.js

```diff
@@ -25,6 +23,14 @@ olcs.core.OlLayerPrimitive = function(layerProjection, scene) {
     primitives: primitives
   };
 
-  this.add(billboards);
+  this.rootCollection_.add(billboards);
+  this.rootCollection_.add(primitives);
+};
+
+
+/**
+ * @return {!Cesium.Primitive}
+ */
+olcs.core.VectorLayerCounterpart.prototype.getRootPrimitive = function() {
```

Comment:
`Primitive` is a `Cesium` concept so I mean having both `Cesium` and `Primitive` sounds redundant.
On the contrary there are several primitives stored in the object and that is why it is important to make clear we are returning the `root` of the Primitive hierarchy.
For example, we also store the shared billboard collection in the object and we may add a getter for it in the future.